rpc/js/src/dial.ts

```diff
@@ -474,6 +474,23 @@ export async function dialWebRTC(
       const offerDesc = await pc.createOffer({});
 
       let iceComplete = false;
+      let numCallUpdates = 0;
+      let maxCallUpdateDuration = 0;
+      let totalCallUpdateDuration = 0;
+
+      pc.addEventListener('iceconnectionstatechange', () => {
+        if (pc.iceConnectionState !== 'completed' || numCallUpdates === 0) {
+          return;
+        }
+        let averageCallUpdateDuration =
+          totalCallUpdateDuration / numCallUpdates;
+        console.debug(
+          `caller update statistics: ` +
+            `num_updates: ${numCallUpdates}, ` +
+            `average_duration: ${averageCallUpdateDuration}ms, ` +
+            `max_duration: ${maxCallUpdateDuration}ms`
+        );
```

Comment:
until we start logging this elsewhere (sentry, etc) - let's group the output to cut down on console noise:

```suggestion
        console.groupCollapsed("Caller update statistics")
        console.table({
            num_updates: numCallUpdates,
            average_duration: `${averageCallUpdateDuration}ms`,
            max_duration: `${maxCallUpdateDuration}ms`
        });
        console.groupEnd()
```

if we don't like the table output we can just keep logs are you have them but in the collapsed group.

Reply:
Ah yeah good idea; didn't know about groups. Couldn't take your suggestion verbatum due to lack of `;`s, but put the same change in.